pkg/web/users.go

```diff
@@ -290,8 +291,66 @@ func (s *Server) DeleteUser(c *gin.Context) {
 
 	c.Negotiate(http.StatusOK, gin.Negotiate{
 		Offered:  []string{binding.MIMEJSON, binding.MIMEHTML},
-		HTMLData: gin.H{"UserID": userID},
+		HTMLData: scene.Scene{"UserID": userID},
 		JSONData: api.Reply{Success: true},
 		HTMLName: "user_delete.html",
 	})
 }
+
+func (s *Server) ChangeUserPassword(c *gin.Context) {
```

Comment:
This handler is the Users API resource action to change a specific users's password.